### PR TITLE
Add missing track events to stats

### LINKED ISSUE #9744: Check the tracking events in Stats
We need to check we're sending all the correct tracking events in the refreshed Stats

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #9744 

I've checked the events being send in Stats and there are some fixes necessary.
* `STATS_VIEW_ALL_ACCESSED` event is sent now when the user opens a `ViewAll` activity.
* `STATS_SINGLE_POST_ACCESSED` event is sent when the user opens the Stats post detail screen
* `STATS_PERIOD_DAYS_ACCESSED`, `STATS_PERIOD_WEEKS_ACCESSED`, `STATS_PERIOD_MONTHS_ACCESSED`, `STATS_PERIOD_YEARS_ACCESSED` all send the same event `stats_period_accessed` so I'm adding a parameter containing the granularity to each of these events

It's hard to test these because the events are fired randomly but you can:
* Go to Stats
* Go to any `View all` screen
* Check that the `stats_view_all_accessed` event was triggered (by debugging or using Stetho/Charles)
* Go to Posts and Pages card
* Click on a Post
* Check that the `stats_single_post_accessed` event was triggered (by debugging or using Stetho/Charles)
* Switch the tabs
* Check that the `stats_period_accessed` is being sent with the parameter `days/weeks/months/years`

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by loremattei** [comment]
I'm moving this to 12.5 since 12.4 has been cut. If this PR has to land 12.4, please feel free to target the release branch and to ping me to build a new beta.

**Review by khaykov — COMMENTED** [review]
All tracking events are getting hit as expected, I only left a single comment :)

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllActivity.kt [hunk 1]
```diff
@@ -26,4 +34,49 @@ class StatsViewAllActivity : AppCompatActivity() {
...
+            selectedTab?.let {
+                intent.putExtra(StatsViewAllFragment.SELECTED_TAB_KEY, selectedTab)
+            }
+            granularity?.let {
+                intent.putExtra(StatsAbstractFragment.ARGS_TIMEFRAME, granularity)
+            }
+            localSiteId?.let {
+                intent.putExtra(WordPress.LOCAL_SITE_ID, localSiteId)
+            }
+            AnalyticsTracker.track(Stat.STATS_VIEW_ALL_ACCESSED)
```
[the remark this answers is not in the document]
Comment (planarvoid) [reply WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllActivity.kt]: Good catch! That was a mistake :-) I've fixed it.

**Review by khaykov — APPROVED** [review]
Looks good 👍

**Pull request merged**